pynio/rest.py

```diff
@@ -9,10 +9,13 @@ def __init__(self, host='127.0.0.1', port=8181, creds=None):
         self._creds = creds or ('User', 'User')
         self._url = 'http://{}:{}/{}'.format(host, port, '{}')
 
-    def _get(self, endpoint, timeout=None):
+    def _get(self, endpoint, timeout=None, data=None):
```

Comment:
I think data should come before timeout to be consistent with the other methods. And for that matter (to be handled later), the other ones should probably be refactored to be 'data' instead of 'config'.